http/server.ts

```diff
@@ -312,10 +310,10 @@ export class Server {
       // Send the response.
       await requestEvent.respondWith(response);
     } catch {
-      // respondWith() fails when the connection has already been closed, or there is some
-      // other error with responding on this connection that prompts us to
-      // close it and open a new connection.
-      return this.#closeHttpConn(httpConn);
+      // `respondWith()` can throw for various reasons, including downstream but also upstream connection errors,
+      // as well as errors thrown during streaming of the response content.
+      // In order to avoid false negatives, we ignore the error here and let `serveHttp` close the connection on
+      // the following iteration if it is in fact a downstream connection error.
```

Comment:
Does it make sense to log something here? Errors that are completely swallowed scare me.

nit: wrap at 80 columns